src/core/rootSaga.js

```diff
@@ -16,5 +17,6 @@ export default function* rootSaga() {
     personListSaga(),
     personDetailsSaga(),
     personCreditsSaga(),
+    watchFetchGenres(),
   ]);
 }
```

Comment:
Where is rest: 
 movieDetailsSaga(),
    movieListSaga(),
    movieCreditsSaga(),
    movieSearchParamsSaga(),
?